data/Signing-Algorithm/PS.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "../../schemas/Signing-Algorithm.json",
+  "Name": "PS",
+  "Implementation Support": "Hyperledger Anoncreds, https://github.com/hyperledger/anoncreds-v2-rs ",
+  "Specification": "PS-Signatures of there own do not have a formal specification, that is included in Anoncreds-V2",
```

Comment:
Could you add also the link here? Then this value is independent from others in this file

Reply:
What link?

Reply:
You are mentioning that it is included in the spec. I think it would be helpful for readers to have a link when you mention it there.